### PR TITLE
[Task][40903] Authentication with firebase

### PR DESCRIPTION
**Pull request by dun99**
https://edu-redmine.sun-asterisk.vn/issues/40903

### CODE REVIEW

**Comment by dun99** [comment]
ready

File: src/components/Header/Header.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            background-color: $black;
+            color: $white;
+        }
+    }
+
+    .ant-layout-header{
+        padding: 0;
+    }
+}
+}
```
Comment (vietnq-1097) [inline src/components/Header/Header.scss:23]: Check tab indent + thêm endline cuối file nhé

File: src/components/LoginForm/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  const handleLogout = () => {
+    dispatch(logoutRequest());
+  };
+
+  useEffect(() => {
+    const unregisterAuthObserver = auth.onAuthStateChanged((user) => {
+      dispatch(setUserCurrent(user));
+    });
+    return () => unregisterAuthObserver();
```
Comment (vietnq-1097) [inline src/components/LoginForm/LoginForm.js:33]:
```suggestion
    unregisterAuthObserver();
```

**Comment by dun99** [comment]
ready

**Pull request merged**